### PR TITLE
Issue #708 Rewording around warning message for config set

### PR DESCRIPTION
**Pull request by praveenkumar**
This patch is only for rewording around config set method.

### CODE REVIEW

File: cmd/minishift/cmd/config/validations.go [hunk 1]
```diff
@@ -38,7 +38,9 @@ func IsValidDriver(string, driver string) error {
 }
 
 func RequiresRestartMsg(string, string) error {
-	fmt.Fprintln(os.Stdout, "To apply the changes, you must delete the current VM with `minishift delete` and start a new VM with `minishift start`.")
+	fmt.Fprintln(os.Stdout, "To apply the changes, "+
+		"you must delete the current VM with `minishift delete` if VM exist "+
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/config/validations.go]: s/exist/exists/

Comment (hferentschik) [inline cmd/minishift/cmd/config/validations.go]: I think we can do better than this. We literally should check whether the machine exists. If so we write print:

"No Minishift VM defined. Settings will be applied on next 'minishift start'"

If the machine does exist:

"Settings are only applied when a new Minishift instance is created. You currently have an existing VM. To apply the changes, you must delete the current VM with `minishift delete` and start a new VM with `minishift start`."

Comment (LalatenduMohanty) [inline cmd/minishift/cmd/config/validations.go]: @hferentschik +1.

Comment (hferentschik) [inline cmd/minishift/cmd/config/validations.go]: There are several ways to go about finding out whether the VM exits.

- Without modifying the current callback signature
  - You can just do what we do in other commands as well. Directly create the api `libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))`
  - We can put something into root.go. In _persistentPreRun_ we determine once for all commands whether the machine exists and set a variable accordingly. This variable can be queried by the other commands. AS a side effect we can remove some VM existence checks and we also get this into one single place. 
- with changing the callback signature
   - You can add additional parameters to the callback method, for example the name and of the VM. This way we stop the proliferation of using the constants file.

Either way, there are several ways to do that.

Comment (LalatenduMohanty) [inline cmd/minishift/cmd/config/validations.go]: I had raised the issue as part of https://github.com/minishift/minishift/issues/471

Comment (thatdocslady) [inline cmd/minishift/cmd/config/validations.go]: So is it safe to review this message now, or will we be changing the command itself to first check if the vm exists?

File: cmd/minishift/cmd/config/validations.go [hunk 2]
```diff
@@ -38,7 +40,19 @@ func IsValidDriver(string, driver string) error {
...
+	defer api.Close()
+
+	_, err := cluster.CheckIfApiExistsAndLoad(api)
+	if err != nil {
+		fmt.Fprintln(os.Stdout, "No Minishift VM exists. Settings will be applied on next `minishift start`")
+	} else {
+		fmt.Fprintln(os.Stdout, "Settings are only applied when a new Minishift instance is created. "+
+			"You currently have an existing VM. "+
+			"To apply the changes, "+
+			"you must delete the current VM with `minishift delete` if VM exist "+
```
Comment (hferentschik) [inline cmd/minishift/cmd/config/validations.go]: no "if VM exists" - we know it does, right?

**Comment by praveenkumar** [comment]
> You can just do what we do in other commands as well. Directly create the api libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))

@hferentschik this is what I chose for this issue right now.
    
> We can put something into root.go. In persistentPreRun we determine once for all commands whether the machine exists and set a variable accordingly. This variable can be queried by the other commands. AS a side effect we can remove some VM existence checks and we also get this into one single place.

@hferentschik As you said we might have to remove some VM existance checks from different command so I think we should do it after our release.

File: cmd/minishift/cmd/config/validations.go (hunk 2 above)
Comment (praveenkumar) [inline cmd/minishift/cmd/config/validations.go]: @hferentschik Right my mistake. Updated.

**Comment by hferentschik** [comment]
Merged. I changed the message and included the name of the setting being set as well.

**Pull request closed without merging**